core/src/bin/operators_and_verifiers.rs

```diff
@@ -15,7 +15,7 @@ use clementine_core::{cli, extended_rpc::ExtendedRpc, servers::create_verifiers_
 #[tokio::main]
 async fn main() {
     let config = cli::get_configuration();
-    let rpc = ExtendedRpc::<bitcoincore_rpc::Client>::new(
+    let _rpc = ExtendedRpc::<bitcoincore_rpc::Client>::new(
```

Comment:
This is not used. Can be completely removed.

Reply:
I think `new` has some side effects inside that make this not removable

Reply:
On my environment, removing this didn't cause any problems. But looks like `servers` got refactored yesterday. There could be another changes in the near future which will require this again. Keeping sounds OK.